src/presentation/components/common/Input/index.tsx

```diff
@@ -30,37 +31,41 @@ function CommonInput(props: CommonInputProps) {
     disabled = false,
     submitButtonValue,
     submitButtonDisabled = false,
+    errorMsg,
   } = props;
+
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     onChange?.(e.target.value);
   };
 
   return (
-    <StCommonInput>
-      <StInputWrapper
-        onSubmit={(e) => {
-          e.preventDefault();
-          onSubmit && onSubmit(e);
-        }}
-        width={width}
-      >
-        <StInput
+    <FormItem value={value ?? ''} errorMsg={errorMsg} maxLength={maxLength}>
+      <StCommonInput>
+        <StInputWrapper
+          onSubmit={(e) => {
+            e.preventDefault();
+            onSubmit?.(e);
+          }}
           width={width}
-          onChange={handleChange}
-          onBlur={onBlur}
-          maxLength={maxLength}
-          placeholder={placeholder || ''}
-          value={value}
-          defaultValue={defaultValue}
-          img={img}
-          disabled={disabled}
-          hasButton={submitButtonValue !== undefined}
-        />
-        {submitButtonValue && (
-          <StSubmitButton disabled={submitButtonDisabled}>{submitButtonValue}</StSubmitButton>
-        )}
-      </StInputWrapper>
-    </StCommonInput>
+        >
+          <StInput
+            width={width}
+            onChange={handleChange}
```

Comment:
onSubmit처럼 여기서 바로 작성해도 좋을 것 같습니다!

```suggestion
            onChange={(e) => onChange?.(e.target.value)}
```

Reply:
렌더될때마다 함수가 새로 생기지 않아도 되어서 기존 방식도 좋을 것 같아용
나중에 함수가 3~4줄 더 길어질 수도 있구용!!

Reply:
@SeojinSeojin 말씀에 동의합니다 !